spec/lib/url_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 RSpec.describe URL, type: :lib do
   before do
+    allow(ApplicationConfig).to receive(:[]).and_call_original
     allow(ApplicationConfig).to receive(:[]).with("APP_PROTOCOL").and_return("https://")
     allow(ApplicationConfig).to receive(:[]).with("APP_DOMAIN").and_return("test.forem.cloud")
```

Comment:
the purpose of this (never called) APP_DOMAIN config value is to contrast that with the app_domain setting in the test for URL.domain, "returns the value of Settings::General"